economics/staking_protocol/main.tex

```diff
@@ -0,0 +1,355 @@
+\documentclass[longbibliography,nofootinbib]{revtex4-1}
+
+\newcommand{\kms}{NuCypher}
+
+\usepackage{listings}
+\usepackage{graphicx}
+\usepackage{amsmath}
+\usepackage{subcaption}
+\usepackage[labelformat=parens,labelsep=quad, skip=3pt]{caption}
+\usepackage[margin=5pt]{subfig}
+\usepackage[usenames]{color}
+
+\renewcommand{\baselinestretch}{1.4}
+\setlength{\parskip}{1em}
+\definecolor{darkgreen}{rgb}{0.00,0.50,0.25}
+\definecolor{darkblue}{rgb}{0.00,0.00,0.67}
+\newcommand{\figref}[1]{Fig.~\ref{#1}}
+\usepackage[breaklinks,pdftitle={NuCypher: a proxy re-encryption network to empower privacy in decentralized systems}, pdfauthor={Arjun Hassard},colorlinks,urlcolor=blue,citecolor=darkgreen,linkcolor=darkblue]{hyperref}
+\graphicspath{{pdf/}}
+
+\usepackage[T1]{fontenc}
+\usepackage{lmodern}
+\lstset{
+    basicstyle=\ttfamily,
+    basewidth={0.5em, 0.5em},
+    columns=fullflexible,
+}
+
+\begin{document}
+
+\title{NuCypher Network: Staking Protocol \& Economics}
+
+\author{Michael Egorov}
+\email{michael@nucypher.com}
+\author{MacLane Wilkison}
+\email{maclane@nucypher.com}
+\author{Arjun Hassard}
+\email{arjun@nucypher.com}
+\affiliation{NuCypher}
+
+\begin{abstract}
+This paper describes the protocol mechanisms which aim to incentivize and sustain reliable supply participation in the NuCypher network, including the regular distribution of a subsidy to actively staking service-providers. The subsidy is generated through the scheduled growth of the native token's circulating supply, following a `two-phase' model designed to maximize service quality, node operation efficiency and network decentralization. The paper also explains the relationship between discretionary stake configurations and subsidy size; in particular, extra incentives to commit to longer staking durations. The rules for stake management, including locking, unwinding, auto-prolonging and withdrawing, are covered in detail, and the impact of these service-provider decisions on the global supply is deduced. Finally, an analysis of historical service-provider behavior in existing staking networks provides an empirical context for design and parameter choices.
```

Comment:
```suggestion
This paper describes the protocol mechanisms which aim to incentivize and sustain reliable supply participation in the NuCypher network, including the regular distribution of a subsidy to actively staking service-providers. The subsidy is generated through the scheduled growth of the native token's circulating supply, following a `two-phase' model designed to maximize service quality, node operation efficiency and network decentralization. The paper also explains the relationship between discretionary stake configurations and subsidy size; in particular, extra incentives to commit to longer staking durations. The rules for stake management, including locking, unwinding, and withdrawing, are covered in detail, and the impact of these service-provider decisions on the global supply is deduced. Finally, an analysis of historical service-provider behavior in existing staking networks provides an empirical context for design and parameter choices.
```
imo remove `auto-prolonging` since it's just the corollary to `unwinding` and it doesn't have an explicit subsection in Section C.